atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -985,7 +1006,17 @@ private static LockAndTimestampServices getLockAndTimestampServices(
             Supplier<RemotingClientConfig> remotingConfigSupplier,
             UserAgent userAgent,
             String timelockNamespace,
-            Optional<ClientLockDiagnosticCollector> lockDiagnosticCollector) {
+            Optional<ClientLockDiagnosticCollector> lockDiagnosticCollector,
+            boolean useDialogue) {
+
+        ConjureClientFactory clientFactory = new ConjureClientFactory(
```

Comment:
I'm assuming you need to wire the flag here?

Reply:
It is: https://github.com/palantir/atlasdb/pull/4706/files/390da9fb497272cb708864adff500acddd9e7098#diff-4d9676a5ce96e75de18356e3d09145f6R1018